metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures ossec"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.5"
+version          "1.0.6"
```

Comment:
Please don't increment versions in PR's. The maintainers will do that when they cut a new release.